orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -706,11 +706,12 @@ def _embedding_hook(module, args, output):
             valid_token = torch.count_nonzero(ebd_input - module.padding_idx)
             total_token = ebd_input.numel()
             embed_density = float(valid_token) / float(total_token) * 100
+            if module not in self._runtime_inspector._embedding_module_to_padding_density_map:
```

Comment:
will this happen?

Reply:
I think it will not. This check is just avoid fail if it happen. Should I delete it?